src/meta/src/hummock/metrics_utils.rs

```diff
@@ -555,36 +551,31 @@ pub fn trigger_write_stop_stats(
     }
 }
 
-pub fn trigger_split_stat(
-    metrics: &MetaMetrics,
-    compaction_group_id: CompactionGroupId,
-    member_table_id_len: usize,
-    branched_ssts: &BTreeMap<
-        // SST object id
-        HummockSstableObjectId,
-        BranchedSstInfo,
-    >,
-) {
-    let group_label = compaction_group_id.to_string();
-    metrics
-        .state_table_count
-        .with_label_values(&[&group_label])
-        .set(member_table_id_len as _);
+pub fn trigger_split_stat(metrics: &MetaMetrics, version: &HummockVersion) {
```

Comment:
build_branched_sst_info need be called whenever a compaction task is reported, which I'm concerned is compute-intensive.
@Li0k @Little-Wallace

Reply:
I'm OK with simply disabling this stats.

Reply:
I have changed to trigger the split stat when we periodically do the checkpoint.